gflownet/policy/multihead_tree.py

```diff
@@ -0,0 +1,502 @@
+from typing import Optional
+
+import torch
+import torch_geometric
+from torch_geometric.data import Batch
+from torch_geometric.nn import global_add_pool
+from torch_geometric.utils import unbatch
+
+from gflownet.envs.tree import Attribute, Stage, Tree
```

Comment:
Rather minor. Similar as for `simple_tree.py` but I will repeat (and adapt) here: Since only the (static) methods `Tree.to_pyg()` and `Tree._find_active()` are used in this class, shall we instead import only these methods instead of the whole class? 

And actually, shall we also remove the underscore of `_find_active` since it is used outside the class, or keep it private? I am not sure of the convention.

Reply:
As above, I renamed the method, and unless you can import some static methods directly I would personally keep it as is.

Reply:
Alright!